src/planscape/plan/models.py

```diff
@@ -0,0 +1,25 @@
+from django.contrib.auth.models import User
+from django.contrib.gis.db import models
+
+
+class Plan(models.Model):
+    """
+    A Plan is associated with one User, the owner, and one Region.  It has a name,
+    status (locked/public), and a geometry representing the planning area.
+    """
+    owner = models.ForeignKey(User, on_delete=models.CASCADE)  # type: ignore
+
+    # The name of the plan.
+    name: models.CharField = models.CharField(max_length=120)
+
+    # The region_name for the plan; should be one of the valid region names in base.region_names.
+    region_name: models.CharField = models.CharField(max_length=120)
+
+    # Whether the plan has been made "public".
+    public: models.BooleanField = models.BooleanField(null=True)
```

Comment:
should we make these defaults false? i.e. a Plan by default is always private and unlocked

Reply:
or should we just have null have the same behavior as false?

Reply:
I've added the default---good call!